config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-0-3.yaml

```diff
@@ -59,15 +61,16 @@ presubmits:
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200929-82b41a1-1.18
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api
-      testgrid-tab-name: pr-apidiff
-  - name: pull-cluster-api-verify
+      testgrid-tab-name: capi-pr-apidiff-release-0-3
```

Comment:
I see you're renaming the PR tabs to be prefixed with capi-. Any particular reason/need for this change?

Reply:
Mostly for consistency, and if we ever also add the providers here, it'll be already separated